src/test/java/com/softserveinc/ita/SearchTest.java

```diff
@@ -18,7 +18,8 @@ public void verifySearchResultsTest() {
                 .setTextInSearchBar(requiredItem)
                 .performSearch();
```

Comment:
after merging from master, it will return SearchResultPage, so it will init your variable for this page

Reply:
i have to merge?